README.md

```diff
@@ -13,7 +13,7 @@ Currently, the following analyzers are included:
 - HTTP <sup>[1]</sup>
 - [IPSec](https://github.com/zeek/spicy-analyzers/tree/main/analyzer/ipsec)
 - LDAP
-- [OpenVPN](https://github.com/zeek/spicy-analyzers/tree/main/analyzer/openvpn)
+- OpenVPN deprecated by <https://github.com/corelight/zeek-spicy-openvpn>
```

Comment:
Let's instead pull this in as a dependency, see comments in https://github.com/zeek/spicy-analyzers/pull/102#pullrequestreview-815950402.